priv/repo/migrations/20170916210952_remove_projects.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Api.Repo.Migrations.RemoveProjects do
+  use Ecto.Migration
+
+  def change do
+    drop table(:projects)
```

Comment:
Shouldn't we be migrating data here? What should we do in prod?

Reply:
The projects are repeated and broken in the platform https://makeorbreak.portosummerofcode.com/admin/projects
I was thinking of throwing them away but I can make some migration script if you think it's worth to keep them.

Reply:
I think a dump is enough to preserve data, in this case. If this has no user-facing use case, I say I'm down for removal.